library/core/src/lib.rs

```diff
@@ -397,7 +398,8 @@ pub mod primitive;
     unused_imports,
     unsafe_op_in_unsafe_fn,
     ambiguous_glob_reexports,
-    deprecated_in_future
+    deprecated_in_future,
+    unreachable_pub
```

Comment:
Uhh, no trailing comma. https://rust-lang.github.io/rustfmt/?version=master&search=#trailing_comma looks promising, but it already should add it for vertical list. Looks like missed case with attributes?

Reply:
https://github.com/rust-lang/rustfmt/issues/6439